### PR TITLE
Fix ambiguous exports

### PR DESCRIPTION
**Pull request by Protryon**
Fixes ambiguous export behavior, #53.

### CODE REVIEW

File: src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java [hunk 1]
```diff
@@ -170,6 +176,9 @@ private ImportExportConnector() {
 				continue; // should become available in later iterations
 			}
 			HashTable<Maybe<Module>, Pair<ExportDeclaration, Variable>> localExported = maybeLocalExported.fromJust();
+			if (localExported.length == 0) {
+				throw new RuntimeException("Ambiguous proxy export");
```
Comment (michaelficarra) [inline src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java]: Can you include the ambiguous name?

File: src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java [hunk 2]
```diff
@@ -636,8 +645,11 @@ public static Script combineModules(@Nonnull BundlerOptions options, @Nonnull Mo
 					if (localSource.isJust()) {
 						objectProperties = objectProperties.cons(new DataProperty(new StaticPropertyName(propertyName), new IdentifierExpression(localExportEntry.right.entries().maybeHead().fromJust().right.right.name)));
 					} else {
-						Pair<Maybe<Module>, Pair<ExportDeclaration, Variable>> remoteSource = localExportEntry.right.entries().maybeHead().fromJust();
-						objectProperties = objectProperties.cons(new DataProperty(new StaticPropertyName(propertyName), new IdentifierExpression(exported.get(remoteSource.left.fromJust()).fromJust().get(inverseExports.get(remoteSource.left.fromJust()).fromJust().get(localExportEntry.right.entries().maybeHead().fromJust().right.right.name).fromJust()).fromJust().entries().maybeHead().fromJust().right.right.name)));
+						Maybe<Pair<Maybe<Module>, Pair<ExportDeclaration, Variable>>> maybeRemoteSource = localExportEntry.right.entries().maybeHead();
+						if (maybeRemoteSource.isJust()) {
```
Comment (michaelficarra) [inline src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java:649]: Wouldn't it be nicer to just `forEach` on `localExportEntry.right.entries().maybeHead()`?

**Review by michaelficarra — APPROVED** [review]
LGTM otherwise.

File: src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java (hunk 2 above)
Comment (Protryon) [reply src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java:649]: Using a lambda for `foreach` would result in having to use the array hack for external mutable variables... java.

**Review by michaelficarra — APPROVED** [review]

**Pull request merged**